sample/shared/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+import org.gradle.kotlin.dsl.sourceSets
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+import org.jetbrains.kotlin.gradle.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    id("com.mikepenz.convention.kotlin-multiplatform")
+    id("com.mikepenz.convention.android-library")
+    id("com.mikepenz.convention.compose")
+    id("com.mikepenz.aboutlibraries.plugin")
+}
+
+kotlin {
+    androidTarget {
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_11)
```

Comment:
can we please go with java 17 which is the java version my convention plugin also applies